lbry/stream/downloader.py

```diff
@@ -48,26 +48,18 @@ async def cached_read_blob(blob_info: 'BlobInfo') -> bytes:
         self.cached_read_blob = cached_read_blob
 
     async def add_fixed_peers(self):
-        def _delayed_add_fixed_peers():
+        async def _add_fixed_peers():
+            await prime_node_with_peer_addresses(self.config.fixed_peers, self.peer_queue)
             self.added_fixed_peers = True
-            self.peer_queue.put_nowait([
-                make_kademlia_peer(None, address, None, tcp_port=port, allow_localhost=True)
-                for address, port in addresses
-            ])
 
         if not self.config.fixed_peers:
             return
-        addresses = [
-            (await resolve_host(url, port, proto='tcp'), port)
-            for url, port in self.config.fixed_peers
-        ]
         if 'dht' in self.config.components_to_skip or not self.node or not \
                 len(self.node.protocol.routing_table.get_peers()) > 0:
             self.fixed_peers_delay = 0.0
         else:
             self.fixed_peers_delay = self.config.fixed_peer_delay
-
-        self.fixed_peers_handle = self.loop.call_later(self.fixed_peers_delay, _delayed_add_fixed_peers)
+        self.fixed_peers_handle = self.loop.call_later(self.fixed_peers_delay, asyncio.create_task, _add_fixed_peers())
```

Comment:
imho it would be cleaner to have ``sleep`` followed by ``_add_peers`` and no cancel handle. But maybe I am missing some important info and the cancelling handle should stay in place. Anyway, it is more asyncio-ish than before.